acm/odh-edge/apps/telemetry/otel-collector.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: cluster-collector
+  namespace: ai-edge-collector
```

Comment:
If this is expected to be a centralized namespace on each edge cluster for all applications, then maybe it should be called `opendatahub-edge-collector` (with a `namespace` field in the `kustomization` file for overriding), wdyt?

Reply:
Yeah I have no strong feelings on the naming so I made this chance.